services/parser-database/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Set base image, the slim one allows us to have a small build size
+FROM python:3.8-slim
+
+# set the working directory in the container
+WORKDIR /code
+
+# Exposes the port required for operation on GKE
+EXPOSE 8082
+
+# Copy dependencies file to container
+COPY requirements.txt .
+
+# Install dependencies using PIP
+RUN pip install -r requirements.txt
+
+# Copy Files to container
+COPY . .
+
+# Run the Flask App
+CMD [ "python", "./app.py" ]
```

Comment:
I think you can avoid the `./` notation and just do `app.py` since it's implicit that you're working on the root folder.